project/detector.py

```diff
@@ -0,0 +1,44 @@
+"""
+Sentiment detector for Tidbits:
+Utilizes the Google Cloud Platform Natural Language Processing API.
+"""
+
+
+class SentimentScore:

```

Comment:
Consider using python3 namedtuple. It is quite convenient for classes that just contain member variables. https://docs.python.org/3/library/collections.html#collections.namedtuple